docs/api/contrib.rst

```diff
@@ -38,6 +39,10 @@ D-adaptation
 .. autoclass:: DAdaptAdamWState
     :members:
 
+Differentially Private SGD
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. autofunction:: dpsgd
+
```

Comment:
Add below
  .. autoclass:: DifferentiallyPrivateAggregateState
    :members:

Reply:
Thanks for letting me know!